pkg/driver_sx126x/patches/0002-src-sx126x-fix-set_sync_word-signature.patch

```diff
@@ -0,0 +1,56 @@
+From 24fda0e786ba055081df06a3120c74f1afc6a94c Mon Sep 17 00:00:00 2001
```

Comment:
Not mandatory, but it would be good to also propose the patch upstream. Even if they don't accept PR from github, at least they will be aware of the problem and might include it in a next release.

Reply:
I removed it since the underlying code is actually handling this smartly already by just modifying the necessary nibbles,